src/components/AddBook.js

```diff
@@ -1,23 +1,29 @@
-import store from '../redux/configureStore';
+import { useDispatch } from 'react-redux';
 import { addBookObject } from '../redux/books/books';
 
-function addABook() {
-  const title = document.getElementById('title');
-  const author = document.getElementById('author');
-  if (title.value !== '' && author.value !== '') {
-    store.dispatch(addBookObject(title.value, author.value));
-  }
-}
-
 function AddBook() {
+  const dispatch = useDispatch();
   return (
     <>
       <form className="addBook">
         ADD NEW BOOK
         <br />
         <input type="text" id="title" placeholder="Book Title" name="title" />
         <input type="text" id="author" placeholder="Author" name="author" />
-        <button type="button" onClick={addABook}>Add Book</button>
+        <button
+          type="button"
+          onClick={() => {
+            const title = document.getElementById('title');
+            const author = document.getElementById('author');
+            if (title.value !== '' && author.value !== '') {
+              dispatch(addBookObject(title.value, author.value));
+              title.value = '';
+              author.value = '';
+            }
+          }}
+        >
+          Add Book
+        </button>
```

Comment:
- [ ] According to the project requirements, when the Add Book button is clicked, the action creator for adding a book should be dispatched. To achieve this, you can create a function that takes the inputs from the state and generate a unique `id` and add them to an object, then dispatches the action creator for adding a book with the object as its argument.